src/index.js

```diff
@@ -237,28 +171,45 @@ const processResults = ({ data, errors }) => {
   return {
     error: err,
     summary: data
-      .map(({ path, url, summary, shortSummary, details, report }) => {
-        const obj = { report, details };
-
-        if (summary) {
-          obj.summary = summary.reduce((acc, item) => {
-            acc[item.id] = Math.round(item.score * 100);
-            return acc;
-          }, {});
-        }
-
-        if (path) {
-          obj.path = path;
-          reports.push(obj);
-          return `Summary for path '${chalk.magenta(path)}': ${shortSummary}`;
-        }
-        if (url) {
-          obj.url = url;
-          reports.push(obj);
-          return `Summary for url '${chalk.magenta(url)}': ${shortSummary}`;
-        }
-        return `${shortSummary}`;
-      })
+      .map(
+        ({
+          path,
+          url,
+          summary,
+          shortSummary,
+          details,
+          report,
+          runtimeError,
+        }) => {
+          const obj = { report, details };
+
+          if (!runtimeError && summary) {
+            obj.summary = summary.reduce((acc, item) => {
+              acc[item.id] = Math.round(item.score * 100);
+              return acc;
+            }, {});
+          }
+
+          if (runtimeError) {
```

Comment:
Send the `runtimeError` off to the Deploy Summary, instead of `shortSumary` (which is undefined)